game/card.go

```diff
@@ -35,6 +35,9 @@ type Card struct {
 	// Properties that are relevant for Lands and other mana producers
 	Colorless         int
 	SacrificesForMana bool
+
+	// Properties that are relevant for Auras
+	HostEntersGraveyardEffect *Effect
```

Comment:
how about calling this "EnchantedPermanentDiesEffect" - that is the wording used on the card itself.